aten/src/ATen/TensorMeta.h

```diff
@@ -14,6 +14,11 @@ struct TensorMeta {
     : sizes(_sizes), options(_options) {}
 };
 
+inline Tensor meta_tensor_from_meta(const TensorMeta& meta) {
+  // TODO: eliminate indirection
+  return at::empty_meta(meta.sizes, meta.options);
+}
+
 inline Tensor tensor_from_meta(const TensorMeta& meta) {
   // TODO: eliminate indirection
```

Comment:
What's the downside of this indirection? I think the wrapper function name makes it a little clearer what the goal is in the codegen.

Reply:
It's actually a different indirection; `at::empty_meta` is making a hop in the dispatcher but it's unnecessary, we could hardcode `at::native::empty_meta` here. I haven't done it yet because I want to keep implementation simple for now.

Reply:
ahhh yep